proxysql/test-proxysql

```diff
@@ -32,28 +32,43 @@ function process_proxysql_output {
 
 WORKDIR_ABS=$(cd ${1:-./build/proxysql}; pwd -P)
 rm -fr ${WORKDIR_ABS}/WORKDIR ${WORKDIR_ABS}/PXB ${WORKDIR_ABS}/PXC
-mkdir -p ${WORKDIR_ABS}/WORKDIR/PROXY ${WORKDIR_ABS}/PXB ${WORKDIR_ABS}/PXC
+mkdir -p ${WORKDIR_ABS}/WORKDIR/PROXY  ${WORKDIR_ABS}/WORKDIR/PXC
 
-tar -C ${WORKDIR_ABS}/PXB --strip-components=1 -zxpf $(ls $WORKDIR_ABS/pxb24.tar.gz | head -1)
-tar -C ${WORKDIR_ABS}/PXC --strip-components=1 -zxpf $(ls $WORKDIR_ABS/pxc57.tar.gz | head -1)
-tar -C ${WORKDIR_ABS}/WORKDIR/PROXY  --strip-components=1 -zxpf $(ls $WORKDIR_ABS/proxysql*.tar.gz | head -1)
+PXC_VERSION="5.7"
+
+if [[ "${PXC_VERSION}" == "8.0" ]]; then
+    sudo wget -q -O ${WORKDIR_ABS}/WORKDIR/Percona-XtraDB-Cluster_8.0.20-11.3_Linux.x86_64.glibc2.12-minimal.tar.gz https://www.percona.com/downloads/Percona-XtraDB-Cluster-LATEST/Percona-XtraDB-Cluster-8.0.20-11.3/binary/tarball/Percona-XtraDB-Cluster_8.0.20-11.3_Linux.x86_64.glibc2.12-minimal.tar.gz
+    sudo tar -C ${WORKDIR_ABS}/WORKDIR/PXC --strip-components=1 -zxpf $(ls $WORKDIR_ABS/WORKDIR/Percona-XtraDB-Cluster_8.0.20-11.3_Linux.x86_64.glibc2.12-minimal.tar.gz | head -1)
+elif [[ "${PXC_VERSION}" == "5.7" ]]; then
+    sudo wget -q -O ${WORKDIR_ABS}/WORKDIR/Percona-XtraDB-Cluster-5.7.31-rel34-45.3.Linux.x86_64.glibc2.12-minimal.tar.gz  https://www.percona.com/downloads/Percona-XtraDB-Cluster-57/Percona-XtraDB-Cluster-5.7.31-31.45-3/binary/tarball/Percona-XtraDB-Cluster-5.7.31-rel34-45.3.Linux.x86_64.glibc2.12-minimal.tar.gz
+    sudo tar -C ${WORKDIR_ABS}/WORKDIR/PXC --strip-components=1 -zxpf $(ls $WORKDIR_ABS/WORKDIR/Percona-XtraDB-Cluster-5.7.31-rel34-45.3.Linux.x86_64.glibc2.12-minimal.tar.gz | head -1)
+    if [ -f /usr/bin/yum ]; then
+        sudo yum remove -y Percona-Server-shared-56-5.6.49-rel89.0.1.el7.x86_64
+        sudo yum install -y percona-xtrabackup-24 Percona-XtraDB-Cluster-57 proxysql2
+    else
+        sudo apt update -y
+        sudo apt install -y percona-xtrabackup-24 percona-xtradb-cluster-client-5.7 proxysql2
+    fi
+fi
 
-PXC57_TAR=$(tar --exclude="*/*" -ztf $WORKDIR_ABS/pxc57.tar.gz)
-mv $WORKDIR_ABS/pxc57.tar.gz $WORKDIR_ABS/WORKDIR/${PXC57_TAR%?}.tar.gz
+tar -C ${WORKDIR_ABS}/WORKDIR/PROXY  --strip-components=1 -zxpf $(ls $WORKDIR_ABS/proxysql*.tar.gz | head -1)
 
-sudo cp ${WORKDIR_ABS}/PXB/bin/* /usr/bin/
-sudo cp ${WORKDIR_ABS}/PXC/bin/* /usr/bin/
+#sudo cp ${WORKDIR_ABS}/WORKDIR/PXC/bin/* /usr/bin/
```

Comment:
Why it's commented but not removed?

Reply:
It is not required, will remove that line.